packages/cli/src/tool-data/user-agent-info.ts

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { injectable } from 'inversify';
+import Puppeteer from 'puppeteer';
+
+@injectable()
+export class UserAgentInfo {
+    constructor(private readonly puppeteer: typeof Puppeteer = Puppeteer) {}
+
+    public async getInfo(): Promise<string> {
+        const browser = await this.puppeteer.launch({ args: ['--disable-dev-shm-usage'] });
+
+        // tslint:disable-next-line:no-unnecessary-local-variable
+        const userAgent = await browser.userAgent();
+
+        await browser.close();
+
+        return userAgent;
+    }
+}
```

Comment:
This is not efficient to launch browser just to get user agent string. We need to get it from where we run scan, but not from UserAgentInfo. No need to have this class.